packages/lib-classifier/src/plugins/drawingTools/components/Polygon/Polygon.js

```diff
@@ -84,7 +83,7 @@ function Polygon({ active, mark, scale, onFinish }) {
               y={point.y}
               fill='currentColor'
               dragMove={point.moveTo}
-              onPointerDown={handleClosePolygon}
+              onPointerUp={handleClosePolygon}
```

Comment:
Previously this overrode `onPointerDown` for draggable elements, which stopped dragging from happening at all. Now it overrides `onPointerUp` which means dragging is never canceled once it has started. 😳

Reply:
Fixed in 4059705.

Reply:
Tested in Chrome and Firefox with a few different drawing tools. Seems to work OK, and dragging is cancelled on pointer up.
https://localhost:8080/?project=eatyourgreens%2F-project-testing-ground&workflow=3370

Reply:
The ellipse tool still seems to work as expected in I Fancy Cats.